RNCamera 1.0.3 crashes on Android after the addition of text recognition.

### Which implementation are you using

RNCamera

### Steps to reproduce
1.  Start out at react-native-camera@1.0.2
2. yarn install react-native-camera@1.0.3
3. Try to use the camera. It will likely crash on android.

### Does it work with Expo Camera?
Yes.

### Expected behaviour
I expect the camera to start up without crashing the app.

### Actual behaviour
Initializing the camera will crash the app on android. Tried to find what caused it by going from commit to commit and found the culprit to be #1388.  Logcat error:

```
03-29 12:15:18.565 27740 27740 E AndroidRuntime: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/internal/zzbko;
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.google.android.gms.vision.text.TextRecognizer$Builder.<init>(Unknown Source)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at org.reactnative.camera.RNCameraView.<init>(RNCameraView.java:83)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at org.reactnative.camera.CameraViewManager.createViewInstance(CameraViewManager.java:53)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at org.reactnative.camera.CameraViewManager.createViewInstance(CameraViewManager.java:16)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.facebook.react.uimanager.ViewManager.createView(ViewManager.java:44)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.facebook.react.uimanager.NativeViewHierarchyManager.createView(NativeViewHierarchyManager.java:224)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.facebook.react.uimanager.UIViewOperationQueue$CreateViewOperation.execute(UIViewOperationQueue.java:153)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.facebook.react.uimanager.UIViewOperationQueue$1.run(UIViewOperationQueue.java:813)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.facebook.react.uimanager.UIViewOperationQueue.flushPendingBatches(UIViewOperationQueue.java:926)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.facebook.react.uimanager.UIViewOperationQueue.access$2100(UIViewOperationQueue.java:47)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.doFrameGuarded(UIViewOperationQueue.java:986)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.facebook.react.uimanager.GuardedFrameCallback.doFrame(GuardedFrameCallback.java:31)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame(ReactChoreographer.java:136)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame(ChoreographerCompat.java:107)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:872)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at android.view.Choreographer.doCallbacks(Choreographer.java:686)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at android.view.Choreographer.doFrame(Choreographer.java:618)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:860)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:751)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:95)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:154)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6119)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
03-29 12:15:18.565 27740 27740 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: com.google.android.gms.internal.zzbko
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at java.lang.VMClassLoader.findLoadedClass(Native Method)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at java.lang.ClassLoader.findLoadedClass(ClassLoader.java:742)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at java.lang.ClassLoader.loadClass(ClassLoader.java:362)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
03-29 12:15:18.565 27740 27740 E AndroidRuntime:        ... 25 more
03-29 12:15:18.565 27740 27740 E AndroidRuntime: Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/common/internal/safeparcel/zza;
```

### Environment
- **Node.js version**: 9.4.0
- **React Native version**: 0.53.3
- **React Native platform + platform version**: Android 7.1.1

### react-native-camera
**Version**: 1.0.3


### Potential solution
I managed to fix this issue by changing my build.gradle file. Seems like the text recognition part isn't in play-services-vision version 10.2.0 which is the default for this package.

Changing from 

```gradle
compile (project(':react-native-camera')) {
    exclude group: "com.google.android.gms"
    compile 'com.android.support:exifinterface:27.+'
    compile ("com.google.android.gms:play-services-vision:10.2.0") {
  	    force = true;
   }
}
```

to 

```gradle
compile (project(':react-native-camera')) {
    exclude group: "com.google.android.gms"
    compile 'com.android.support:exifinterface:27.+'
    compile ("com.google.android.gms:play-services-vision:12.+")
}
```

fixed the issue for me.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -16,7 +16,7 @@ apply plugin: 'com.android.library'
 def DEFAULT_COMPILE_SDK_VERSION             = 26
 def DEFAULT_BUILD_TOOLS_VERSION             = "26.0.2"
 def DEFAULT_TARGET_SDK_VERSION              = 26
-def DEFAULT_GOOGLE_PLAY_SERVICES_VERSION    = "10.2.0"
+def DEFAULT_GOOGLE_PLAY_SERVICES_VERSION    = "12.0.1"
 def DEFAULT_SUPPORT_LIBRARY_VERSION         = "27.1.0"
 
 android {

